strings/ctype-bin.c

```diff
@@ -199,7 +199,7 @@ static int my_strnncollsp_8bit_bin(CHARSET_INFO * cs __attribute__((unused)),
 {
   const uchar *end;
   size_t length;
-
```

Comment:
`/server/strings/ctype-bin.c:203:10: runtime error: applying zero offset to null pointer`